.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: |
+          yarn global add lerna
+          # approach taken from https://github.com/actions/setup-node/issues/85
+          echo "$(yarn global bin)" >> $GITHUB_PATH
+      - run: lerna bootstrap
+      - run: lerna run test
+
+name: Build
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    needs: [test]
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install
+      - run: yarn build:dev-ui
+
+name: Deploy Developer UI
+on:
+  push:
+    branches:
+      - master
+      - develop
+    # https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths
+    paths:
+      - "packages/safe-apps-developer-ui/**"
+  pull_request:
+    branches:
+      - master
+  # Launches build when release is published
+  release:
+      types: [published]
+jobs:
+  deploy:
+    name: Deploy
+    needs: [build]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Download website
+        uses: actions/download-artifact@v2
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: 'Deploy to S3: Develop'
+        if: github.ref == 'refs/heads/develop'
+        run: aws s3 sync build/safe-apps-developer-ui s3://${{ secrets.AWS_DEV_BUCKET_NAME }}/safe-apps-developer-ui --delete
+
+      - name: 'Deploy to S3: Staging'
+        if: github.ref == 'refs/heads/master'
+        run: aws s3 sync build/safe-apps-developer-ui s3://${{ secrets.AWS_STAGING_BUCKET_NAME }}/current/safe-apps-developer-ui --delete
+
+      - name: 'Production deployment: Upload release build files to be deployed'
+        if: startsWith(github.ref, 'refs/tags/v')
+        run: aws s3 sync build s3://${{ env.STAGING_BUCKET_NAME }}/releases/${{ github.event.release.tag_name }}/safe-apps-developer-ui --delete
```

Comment:
same here, we should update the current build (all the packages, not a specific one in this case) to S3